### ISSUE TITLE
Convert price widget to ocmod

### ISSUE THREAD

**Issue by KFoobar** [sveawebpay/opencart-module#15]

**Comment by MikhailShemelin** [comment]
Please check https://github.com/sveawebpay/opencart-module/pull/18

**Comment by KFoobar** [comment]
@MikhailShemelin Thank you, we are rebuilding the price widget, but i will check your PR and review what we can use.

**Comment by MikhailShemelin** [comment]
@KFoobar Then you could also consider an idea of implementing it through a separate controller etc. 
Similar to this https://github.com/sveawebpay/opencart-module/pull/19

**Comment by KFoobar** [comment]
@MikhailShemelin Yes, I agree with you on that, that's one of the reasons for the rebuild, trying to minimize the code in the ocmod file and making it reusable.
